[DECLUTTER FAQ] re-write consolidation (#567)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

@[youtube](k4VzJ6dUT9I)

::::details
### Can I consolidate anonset coins?

It is advisable to limit the recombining of <img src="/ShieldCheckmark.png" alt="checkmark" class="shield" /> <img src="/ShieldGreen.png" alt="green" class="shield" /> <img src="/ShieldYellow.png" alt="yellow" class="shield" /> mixed coins because it can only decrease the privacy of these coins.

> This reveals that all the consolidated UTXOs are controlled by one entity, and this was previously not known.

Reviewer: ```suggestion
This reveals that all the consolidated UTXOs are controlled by one entity, which was not known before the consolidation.
```

> That said, if you combine less than 10 BTC, you might not reveal your pre-CoinJoin transaction history, especially when you did several re-mixes.

Reviewer: 10 BTC??
```suggestion
That said, if you combine less than 1 BTC, you might not reveal your pre-CoinJoin transaction history, especially if you did several re-mixes.
```

Author: This comes from [the reddit post](https://old.reddit.com/r/WasabiWallet/comments/avxbjy/combining_mixed_coins_privacy_megathread/) and from a conversation with nopara. 

Though I am hesitant with this too, I believe we should instead have an arbitrary non-specific value here...

```suggestion
That said, if you combine only a couple of mixed coisn, you might not reveal your pre-CoinJoin transaction history, especially when you did several re-mixes.
```
